qiskit/transpiler/passes/utils/gate_direction.py

```diff
@@ -169,6 +176,16 @@ def run(self, dag):
                             "The circuit requires a connection between physical "
                             "qubits %s and %s for cx" % (physical_q0, physical_q1)
                         )
+                elif node.name == "cz":
+                    if (physical_q0, physical_q1) in self.target["cz"]:
+                        continue
+                    if (physical_q1, physical_q0) in self.target["cz"]:
+                        dag.substitute_node_with_dag(node, self._cz_dag)
```

Comment:
I would hope that a `Target` that defines `cz` would add the definition for both directions since it is a symmetric gate, but I guess it doesn't hurt to have this check just in case and ensure the correct behavior.